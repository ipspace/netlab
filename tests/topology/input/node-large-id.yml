#
# Adjust standard address pools to allow large node IDs
#
addressing:
  loopback.ipv4: 10.0.0.0/22
  mgmt.ipv4: 192.168.64.0/22

defaults.device: none
defaults.const.MAX_NODE_ID: 1024

nodes:
  r1:
    id: 7
  r2:
    id: 300
  r3:

links:
- name: Standard LAN link (happens to be v6only)
  r1:
  r2:
  r3:
