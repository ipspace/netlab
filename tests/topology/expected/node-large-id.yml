input:
- topology/input/node-large-id.yml
- package:topology-defaults.yml
links:
- _linkname: links[1]
  bridge: input_1
  interfaces:
  - ifindex: 1
    ifname: eth1
    ipv4: 172.16.0.1/24
    node: r1
  - ifindex: 1
    ifname: eth1
    ipv4: 172.16.0.2/24
    node: r2
  - ifindex: 1
    ifname: eth1
    ipv4: 172.16.0.3/24
    node: r3
  linkindex: 1
  name: Standard LAN link (happens to be v6only)
  node_count: 3
  prefix:
    ipv4: 172.16.0.0/24
  type: lan
name: input
nodes:
  r1:
    af:
      ipv4: true
    box: none
    device: none
    id: 7
    interfaces:
    - bridge: input_1
      ifindex: 1
      ifname: eth1
      ipv4: 172.16.0.1/24
      linkindex: 1
      name: Standard LAN link (happens to be v6only)
      neighbors:
      - ifname: eth1
        ipv4: 172.16.0.2/24
        node: r2
      - ifname: eth1
        ipv4: 172.16.0.3/24
        node: r3
      type: lan
    loopback:
      ifindex: 0
      ifname: Loopback0
      ipv4: 10.0.0.7/32
      neighbors: []
      type: loopback
      virtual_interface: true
    mgmt:
      ifname: eth0
      ipv4: 192.168.64.107
      mac: ca:fe:00:07:00:00
    name: r1
  r2:
    af:
      ipv4: true
    box: none
    device: none
    id: 300
    interfaces:
    - bridge: input_1
      ifindex: 1
      ifname: eth1
      ipv4: 172.16.0.2/24
      linkindex: 1
      name: Standard LAN link (happens to be v6only)
      neighbors:
      - ifname: eth1
        ipv4: 172.16.0.1/24
        node: r1
      - ifname: eth1
        ipv4: 172.16.0.3/24
        node: r3
      type: lan
    loopback:
      ifindex: 0
      ifname: Loopback0
      ipv4: 10.0.1.44/32
      neighbors: []
      type: loopback
      virtual_interface: true
    mgmt:
      ifname: eth0
      ipv4: 192.168.65.144
      mac: ca:fe:00:00:01:2c
    name: r2
  r3:
    af:
      ipv4: true
    box: none
    device: none
    id: 1
    interfaces:
    - bridge: input_1
      ifindex: 1
      ifname: eth1
      ipv4: 172.16.0.3/24
      linkindex: 1
      name: Standard LAN link (happens to be v6only)
      neighbors:
      - ifname: eth1
        ipv4: 172.16.0.1/24
        node: r1
      - ifname: eth1
        ipv4: 172.16.0.2/24
        node: r2
      type: lan
    loopback:
      ifindex: 0
      ifname: Loopback0
      ipv4: 10.0.0.1/32
      neighbors: []
      type: loopback
      virtual_interface: true
    mgmt:
      ifname: eth0
      ipv4: 192.168.64.101
      mac: ca:fe:00:01:00:00
    name: r3
provider: libvirt
